packages/manager/apps/pci-gateway/src/hooks/useDatagridColumn.tsx

```diff
@@ -50,12 +50,7 @@ export const useDatagridColumn = (privateNetworkUrl: string) => {
     {
       id: 'networksConnected',
       cell: (props: Gateway) => (
-        <OsdsLink
-          href={privateNetworkUrl}
```

Comment:
I don't see these values being used, be sure to clean unused varialbes and imports when possible